docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx
+sphinx_glpi_theme
```

Comment:
Can update this to the requriements from the btrdb-python repo.

Makefile: https://github.com/PingThingsIO/btrdb-python/blob/master/docs/Makefile
Req.txt: https://github.com/PingThingsIO/btrdb-python/blob/master/docs/requirements.txt